extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -57,9 +57,12 @@
   border-radius: 50% !important;
   box-sizing: border-box !important;
   position: relative !important;
+  pointer-events: auto !important;
+
 
   /* for svg backgrounds: */
   background: center center no-repeat !important;
+  display: flex !important;
```

Comment:
Is there any reason why you have to use this display flex?

Reply:
Removed it and it works fine so I'll take it out!